### DISCUSSION TITLE
New architecture support for fabric

### DISCUSSION THREAD

**Discussion by BraveEvidence** [iddan/react-native-canvas#334] [Ideas]

**Comment by BraveEvidence** [comment]
This might help https://www.youtube.com/watch?v=wKr1kxNccgo for now

**Comment by csulit** [comment]
It's running fine with the new architecture, but there is a strange warning in Android.

`(NOBRIDGE) WARN  Received unexpected message {"id": 0.3657206053889037, "meta": {}, "type": "json"}`
